nf_core/subworkflows/subworkflows_command.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import shutil
+from pathlib import Path
+
+from nf_core.components.components_command import ComponentCommand
+
+log = logging.getLogger(__name__)
+
+
+class SubworkflowCommand(ComponentCommand):
+    """
+    Base class for the 'nf-core subworkflows' commands
+    """
+
+    def __init__(self, dir, remote_url=None, branch=None, no_pull=False, hide_progress=False):
+        self.default_subworkflows_path = Path("subworkflows", "nf-core")
+        self.default_subworkflows_tests_path = Path("tests", "subworkflows", "nf-core")
+        super().__init__("subworkflows", dir, remote_url, branch, no_pull, hide_progress)
+
+    def get_subworkflows_clone_subworkflows(self):
+        """
+        Get the subworkflows repository available in a clone of nf-core/subworkflows
+        """
+        subworkflow_base_path = Path(self.dir, self.default_subworkflows_path)
+        return [
+            str(Path(dir).relative_to(subworkflow_base_path))
+            for dir, _, files in os.walk(subworkflow_base_path)
+            if "main.nf" in files
+        ]
+
+    def get_local_subworkflows(self):
```

Comment:
Same as above

Reply:
This would be needed in `lint` for `subworkflows` if implemented same way as `modules lint`.